bot.py

```diff
@@ -389,6 +391,32 @@ def __init__(self, *args, choices=None, opts_to_inv=None, **kwargs):
         )
         self.add_item(select_menu)
 
+class EmojiSyncView(discord.ui.View):
+    # Might be able to delete guild reference
+    def __init__(self, emoji, mod_type: str, old_name=None, *args, **kwargs):
+        super().__init__(timeout=None, *args, **kwargs)
+        self.emoji = emoji
+        self.old_name = old_name
+        
+        # Mod type will be a String of 'Add', 'Del', or 'Name' 
+        self.mod_type = mod_type
+
+    @discord.ui.button(label='Accept', style=discord.ButtonStyle.green)
+    async def accept_callback(self, button, interaction: discord.Interaction):
+        if self.mod_type == 'Add':
+            await sync_add(bot=bot, emoji=self.emoji)
+        elif self.mod_type == 'Del':
+            await sync_delete(bot=bot, emoji=self.emoji)
+        else:
+            await sync_name(bot=bot, old_name=self.old_name, new_emoji=self.emoji)
+
+        await interaction.response.edit_message('Okay! I will sync this now.', view=None, delete_after=60)
```

Comment:
Interacting with "accept" button performs required actions but states "this interaction failed" and throws this error in console:

```
Ignoring exception in view <EmojiSyncView timeout=None children=2> for item <Button style=<ButtonStyle.success: 3> url=None disabled=False label='Accept' emoji=None row=None>:
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/discord/ui/view.py", line 375, in _scheduled_task
    await item.callback(interaction)
  File "/PittBOT/bot.py", line 413, in accept_callback
    await interaction.response.edit_message('Okay! I will sync this now.', view=None, delete_after=60)
TypeError: InteractionResponse.edit_message() takes 1 positional argument but 2 positional arguments (and 2 keyword-only arguments) were given
```

Reply:
This is a really easy fix. Put `content=` before the string message. `edit_message()` does not accept content as a positional argument, you have to use the keyword.